README.md

```diff
@@ -51,10 +42,57 @@ plug Plug.Session,
   key_iterations: 1000,
   key_length: 64,
   key_digest: :sha,
-  serializer: Poison
+  serializer: Poison # see serializer details below
 end
 ```
 
+#### Set up a serializer
+
+Plug & Rails must use the same strategy for serializing cookie data.
+
+- __JSON__: Configure your Rails app to serialize with JSON:
+
+  ```ruby
+  Rails.application.config.action_dispatch.cookies_serializer = :json
+  ```
+
+  Then, get a JSON serializer and pass it to `Plug.Session`. For example, add `Poison` to your dependencies, then:
+
+  ```elixir
+  plug Plug.Session,
+    store: PlugRailsCookieSessionStore,
+    # ... see encryption config above
+    serializer: Poison
+  end
+  ```  
+
+- __Marshal__: By default, Rails serializes cookie data with Ruby's [`Marshal` library](http://ruby-doc.org/core-2.3.0/Marshal.html). You can deserialize this by adding [`ExMarshal`](https://hex.pm/packages/ex_marshal) to your project and defining a serializer module:
```

Comment:
Oh, I guess it's "default" in ours because they started in older Rails versions, but I see that new Rails app have the `:json` configuration in an initializer. I'll update this with that information!